packages/pwa-kit-dev/src/configs/webpack/overrides-plugin.test.js

```diff
@@ -128,101 +126,222 @@ describe('overrides plugin', () => {
+        expect(callback).toHaveBeenCalled()
+        expect(resolver.doResolve).toHaveBeenCalledWith(
+            null,
+            createRequestContextWith({
+                path: path.join(REWRITE_DIR, REQUEST_PATH + `/index.jsx`),
+                request: `${EXTENDS_TARGET}/${REQUEST_PATH}`
+            }),
+            expect.anything(),
+            expect.anything(),
+            expect.anything()
+        )
+    })
+
+    test('a nested overrides folder path/to/overrides resolves correctly', () => {
+        const REQUEST_PATH = 'exists'
+        const REQUEST_EXTENSION = '.jsx'
+        const testRequestContext = createRequestContextWith({
+            path: path.join('./', 'node_modules', EXTENDS_TARGET),
+            request: `${EXTENDS_TARGET}/${REQUEST_PATH}`
+        })
 
-    test('projectDir is normalized with forward slashes', () => {})
+        const {resolver, callback} = setupResolverAndCallback(null, testRequestContext, null, {})
 
-    test('a nested overrides folder path/to/overrides resolves correctly', () => {})
+        const nestedOverridesOptions = {
+            overridesDir: '/path/to/overrides',
+            extends: ['@salesforce/retail-react-app'],
+            projectDir: PROJECT_DIR
+        }
+
+        const overridesResolver = new OverridesResolverPlugin(nestedOverridesOptions)
+        overridesResolver.handleHook(testRequestContext, {}, callback, resolver)
+
+        const nestedRewriteDir = 'src/configs/webpack/test/path/to/overrides'
+
+        expect(callback).toHaveBeenCalled()
+        expect(resolver.doResolve).toHaveBeenCalledWith(
+            null,
+            createRequestContextWith({
+                path: path.join(nestedRewriteDir, REQUEST_PATH + REQUEST_EXTENSION),
+                request: `${EXTENDS_TARGET}/${REQUEST_PATH}`
+            }),
+            expect.anything(),
+            expect.anything(),
+            expect.anything()
+        )
+    })
+})
+
+describe('OverridePlugin.isFromExtends Windows', () => {
+    const windowsOptions = {
+        overridesDir: '\\overrides',
+        extends: ['@salesforce/retail-react-app'],
+        projectDir: `src\\configs\\webpack\\test`
+    }
+
+    const plugin = new OverridesResolverPlugin(windowsOptions)
+
+    const cases = [
+        {
+            name: 'Import from a package in extends',
+            request: '@salesforce/retail-react-app/exists',
+            filepath: './node_modules/@salesforce/retail-react-app',
+            result: true
+        },
+        {
+            name: 'Import from a package not in extends',
+            request: '@salesforce/express-minimal/notExists',
+            filepath: './node_modules/@salesforce/retail-react-app',
+            result: false
+        },
+        {
+            name: 'Do not trigger override if filepath of issuer contains overrides directory',
+            request: '@salesforce/retail-react-app/exists',
+            filepath: 'src/configs/webpack/test/overrides/exists',
```

Comment:
@kevinxh @bfeister I added a starting point that applies the suggestion from @kevinxh 

I am finding that since we are normalizing filepaths inside `isFromExtends`, the expected filepath which in this case should be `src\\configs\\webpack\\test\\overrides\\exists` will only return what we expect on Windows. On my mac, this test passes if I set `filepath: 'src/configs/webpack/test/overrides/exists'` instead

Reply:
There are now windows specific and posix specific test cases